### PR TITLE
PMM-6994 add alerts table

### PR DESCRIPTION
**Pull request by nicolalamacchia**
https://github.com/Percona-Lab/pmm-submodules/pull/1299

### CODE REVIEW

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    activeSince: active_since ? moment(active_since).format('YYYY-MM-DD HH:mm:ss.SSS') : '',
+    labels: formatLabels(labels),
+    severity: AlertRuleSeverity[severity],
+    status: AlertStatus[status],
+    summary,
+    lastNotified: last_notified ? moment(last_notified).format('YYYY-MM-DD HH:mm:ss.SSS') : '',
+  };
+};
+
+export const formatAlerts = (rules: AlertsListResponseAlert[]): Alert[] => rules.map(formatAlert);
```
Comment (atymchuk) [inline public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts:28]:
```suggestion
export const formatAlerts = (alerts: AlertsListResponseAlert[]): Alert[] => alerts.map(formatAlert);
```
The parameter name should be `alerts`, no?

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
```
Comment (atymchuk) [inline public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts:5]:
```suggestion
export const getStyles = (theme: GrafanaTheme) => {
```
No need to use the factory as per https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
```
Comment (atymchuk) [inline public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts:3]:
```suggestion
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts (hunk 1 above)
Comment (nicolalamacchia) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts:28]: oops, bad copy/past, fixed

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts (hunk 2 above)
Comment (nicolalamacchia) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts:5]: Fixed and fixed in `AlertRulesTable.styles.ts` as well

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts (hunk 3 above)
Comment (nicolalamacchia) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts]: Fixed and fixed in `AlertRulesTable.styles.ts` as well

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts (hunk 1 above)
Comment (atymchuk) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.utils.ts:28]: LOL: copy & pasta! I'm missing pizza as well :)

**Review by atymchuk — DISMISSED** [review]

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      } as Column,
+      {
+        Header: stateColumn,
+        accessor: 'status',
+        width: '5%',
+      } as Column,
+      {
+        Header: labelsColumn,
+        accessor: ({ labels }: Alert) => (
+          <div className={style.labelsWrapper}>
```
Comment (tiagomotasantos) [inline public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx:60]: nit: move this to a separate component?

**Review by tiagomotasantos — DISMISSED** [review]

File: public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx (hunk 4 above)
Comment (nicolalamacchia) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx:60]: It's super small and doesn't affect readability too much, let's leave it as it is for now :)

Comment (nicolalamacchia) [reply public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx:60]: It's super small and doesn't affect readability too much, let's leave it as it is for now :)

**Review by tiagomotasantos — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**